CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - On Wayland, fix cursor icon updates on window borders when using CSD.
+- On Wayland, under mutter(GNOME Wayland), fix CSD being behind the status bar, when stating window in maximized mode.
```

Comment:
Is this a typo for "starting" (missing "r") ?

Reply:
Yes.

Reply:
I had to force push, because I have to alter commit message(the same typo).